src/client/components/ui/Carousel/Controls.module.scss

```
@@ -19,4 +20,12 @@
   &Next {
     right: 0.5rem;
   }
+  @supports selector(&:focus-visible) {
+    &:focus {
+      box-shadow:0 0 0 4px utilities.$color-background-primary-alternative;
+    }
+    &:focus-visible {
+      box-shadow:0 0 0 4px utilities.$color-background-primary-alternative;
+    }
+  }
```

Je crois que le morceau de code serait plutôt : 

```suggestion
   &:focus {
      box-shadow:0 0 0 4px utilities.$color-background-primary-alternative;
    }

  @supports selector(&:focus-visible) {
    &:focus {
      box-shadow: revert;
    }
    &:focus-visible {
      box-shadow:0 0 0 4px utilities.$color-background-primary-alternative;
    }
  }
```

pour définir par défaut sur le pseudo-selecteur `:focus` (au cas où le `:focus-visible` n'est pas pris en charge) 

et revert / annuler cette règle si on peut la définir plus précisément grâce à un navigateur qui prend en charge `:focus-visible` )